packages/firebase_in_app_messaging/firebase_in_app_messaging_platform_interface/lib/src/platform_interface/platform_interface_firebase_in_app_messaging.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: require_trailing_commas
```

Comment:
```suggestion
```

Reply:
There's a couple other `// ignore_for_file: require_trailing_commas` in some other files too that we should probably remove